test/training.jl

```diff
@@ -0,0 +1,10 @@
+using Base.Test
```

Comment:
Maybe move [this testset](https://github.com/malmaud/TensorFlow.jl/blob/master/test/transformations.jl#L141-L165) here as well